core/src/test/java/io/questdb/test/cutlass/http/IODispatcherTest.java

```diff
@@ -4421,6 +4421,34 @@ public void testJsonQueryMultipleRowsLimitColumns() throws Exception {
         );
     }
 
+    @Test
+    public void testJsonQueryComplexeColumnNames() throws Exception {
+        testJsonQuery(
```

Comment:
this is an old-style test, please look at the newer examples...  i will follow up with an example

Reply:
`testJsonTableReferenceOutOfDate`